scripts/DolbyIOManager.cs

```diff
@@ -24,6 +30,42 @@ public class DolbyIOManager : MonoBehaviour
         [Tooltip("Indicates if the DolbyIOManager should automatically close the session on application quit.")]
         public bool AutoCloseSession = true;
 
+        /// <summary>
+        /// For convenience during early development and prototyping, a GetToken (linked) method is provided for you to 
+        /// acquire a client access token directly from the application. However, please note Dolby does not recommend 
+        /// using this mechanism in the production software for (security best practices)[https://docs.dolby.io/communications-apis/docs/guides-client-authentication] reasons.
+        /// </summary>
+        /// <param name="key">The customer key.</param>
```

Comment:
Let's go with "The App Key" and "The App Secret" to be consistent with what we show in the dashboard.

Reply:
done